core/experiments/manager.php

```diff
@@ -155,6 +155,19 @@ public function add_feature( array $options ) {
 		return $experimental_data;
 	}
 
+	private function install_compare( $version ) {
+		$installs_history = Upgrade_Manager::get_installs_history();
+
+		end( $installs_history );
```

Comment:
does it keep the pointer on the last index after calling `Upgrade_Manager::get_installs_history()` again? (in other words - does `get_installs_history` return the same reference?)

i.e.:

```PHP
$installs_history = Upgrade_Manager::get_installs_history();
end( $installs_history ); // last

// ... 

$installs_history_2 = Upgrade_Manager::get_installs_history();
key($installs_history_2; // still last?
```

Reply:
No. The pointer is on the variable.